composer.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "php-tuf/drupal-project",
-    "description": "Project template for Drupal projects with a relocated document root",
+    "description": "Intended for testing drupal.org TUF staging setup. Project template for Drupal projects with a relocated document root.",
```

Comment:
I just added this commit because I didn't know why the workflow wasn't being triggered(it was disabled).